code/modules/jobs/job_types/assistant.dm

```diff
@@ -63,5 +63,5 @@ Assistant
 
 	// This outfit is used by the assets SS, which is ran before the atoms SS
 	if (SSatoms.initialized == INITIALIZATION_INSSATOMS)
-	//	H.w_uniform?.update_greyscale()
+		H.w_uniform?.update_greyscale()
```

Comment:
Why is this uncommented out?

Reply:
i commented it out when i ported it because we didn't have gags but now we do